genesis/checkpoints.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package genesis
+
+import (
+	"encoding/json"
+	"fmt"
+
+	_ "embed"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/set"
+)
+
+var (
+	//go:embed checkpoints.json
+	checkpointsPerNetworkJSON []byte
+
+	checkpointsPerNetwork map[string]map[ids.ID]set.Set[ids.ID]
+)
+
+func init() {
+	if err := json.Unmarshal(checkpointsPerNetworkJSON, &checkpointsPerNetwork); err != nil {
+		panic(fmt.Sprintf("failed to decode checkpoints.json %v", err))
```

Comment:
Consistency w/ other errors in this file:
```suggestion
		panic(fmt.Sprintf("failed to decode checkpoints.json: %v", err))
```